Medicare and Medicaid/README.md

```diff
@@ -0,0 +1,37 @@
+
+
+In the imputation process, we aimed at targeting total benefit expenditures, excluding administrative expenses, for non-institutionalized population. We obtain enrollee and benefit targets from [Medicaid Actuary Report](https://www.medicaid.gov/medicaid/finance/actuarial-report/index.html) and [Medicare Trustee's Report](https://www.cms.gov/Research-Statistics-Data-and-Systems/Statistics-Trends-and-Reports/ReportsTrustFunds/index.html)
+
+## Total number of recipients
+
+MEPS has already imputed coverage and benefit before distributing the datasets with their household and provider components in the survey. Thus, the number of recipients are fairly close to official reports for both programs. Medicare annual report claims it covered 53.8 million people in 2014, and MEPS Medicare recipients sum up to about 49.6 million, which is not far away since MEPS doesn�t cover institutionalized population. CPS Medicare section also has about 48.9 million recipients, so there�s not much need for imputation. But Medicaid has a slightly different story. Medicaid covers about 64 millions beneficiaries (point in time during FY 2014), while MEPS has about 74 million and CPS has about 54 million. MEPS seems to cover more people who enrolled for a short period of a year, so the total recipients are higher. But CPS is under-reported compared to Medicare. Again, if consider the population living inside institutions, CPS is not too far away. At this moment, we leave CPS coverage information as it is.
+
+## Benefit
+
+The amount of benefit is very pertinent to individual�s health condition, age, close to death, income etc. But information like how close to death is not observable, and some other information like health condition is available in MEPS but not so in CPS. Therefore, we made a compromise with the match and only execute the matching procedure based on variables available in both datasets. In current routine, we used age, gender, census region and income to match MEPS beneficiary to CPS. 
+
+Specifically, for each CPS beneficiary, we find a pool of donors from MEPS defined by following standards:
+
+- age within plus or minus 2 range
+- same gender
+- same census region
+- income within plus or minus 100 range
+
+Then we use a random number generator to pick one donor from the pool defined above, and assign this donor�s benefit amount to the corresponding CPS beneficiary. This set of standards work well for the vast majority of beneficiaries in CPS. However there�re several scenarios the method doesn�t work. First, CPS has beneficiaries older than 85, but the max age in MEPS is 85. Thus all 85 or older CPS medicare/medicaid beneficiary share the same group of donor from MEPS who are labeled at 85 years old. Second, certain age range of donors in MEPS don�t have income fell in the given range. Current routine use the donor with closest income in the given age range. 
```

Comment:
> Second, certain age range of donors in MEPS donÕt have income fell in the given range. Current routine use the donor with closest income in the given age range. 

Not sure I follow this: is it that some CPS beneficiaries don't have a corresponding MEPS donor, so you keep the age/gender/region constraints but relax the income constraint to find the donor with the closest income?